web/src/lib/components/shared-components/immich-thumbnail.svelte

```diff
@@ -259,17 +273,21 @@
 
 		{#if mouseOver && asset.type === AssetTypeEnum.Video}
 			<div class="absolute w-full h-full top-0" on:mouseenter={loadVideoData}>
+				{#if videoUrl}
 				<video
 					muted
 					autoplay
 					preload="none"
 					class="h-full object-cover"
 					width="250px"
 					style:width={`${thumbnailSize}px`}
+					on:canplay={handleCanPlay}
 					bind:this={videoPlayerNode}
 				>
+					<source src="{videoUrl}" type="{asset.mimeType}" />
```

Comment:
The problem is that you are passing in `mimeType` if you uploaded the video from iPhone, the mimeType of the original file will be `video/quicktime` and browser doesn't support playing this mime type. So you will need to either hardcode `video/mp4` or check if the video is not `video/mp4` original;y and it has `encodedVideoPath` then put the type as `video/mp4`

Reply:
For now, I use `video/mp4` in hard code because we assume all non-h264 video types is transcoded to h264